assignment2/Lexic.txt

```diff
@@ -0,0 +1,29 @@
+Alphabet:
+	a) A-Z (upper case letters of the English alphabet)
+	b) a-z (lower case letters of the English alphabet)
+	c) 0-9 (digits)
+	d) _ (underscore character)
+
+Lexic:
+	a) Special symbols:
+		• operators: +, -, *, /, =, ==, >, >=, <, <=, !=
+		• separators: {}, (), [], ;, space, :
+		• reserved words: begin, end int, char, string, array, if, then, else, for, while, read, write, return
+	b) Identifiers: 
+		A sequence of letters, digits or “_”, such that the first character is a letter
+		• <identifier> ::= <letter> | <identifier><letter> | <identifier>_ | <identifier><digit>
```

Comment:
for the 3rd option shouldn't it be something like < identifier>_<identifier> to write smth like abc_abc or abc_ab2